META.json

```diff
@@ -2,15 +2,15 @@
    "name": "citus",
    "abstract": "Scalable PostgreSQL for real-time workloads",
    "description": "Citus horizontally scales PostgreSQL across commodity servers using sharding and replication. Its query engine parallelizes incoming SQL queries across these servers to enable real-time responses on large datasets.",
-   "version": "9.2.4",
+   "version": "9.3.0",
    "maintainer": "\"Citus Data\" <packaging@citusdata.com>",
    "license": "agpl_3",
    "provides": {
       "citus": {
          "abstract": "Citus Distributed Database",
-         "file": "src/backend/distributed/sql/citus--9.1-1--9.2-1.sql",
+         "file": "src/backend/distributed/sql/citus--9.3-2--9.4-1.sql",
```

Comment:
At some point we should stop caring about this field I guess.

Should we just place `citus.so` her and stop modifying it maybe?

Reply:
or readme.md can even make more sense

Reply:
but we don't have readme there, citus.so makes sense